app/src/main/AndroidManifest.xml

```diff
@@ -38,16 +39,22 @@
             android:exported="true">
         </service>
 
-        <receiver android:name=".receivers.WiFiStatusReceiver">
+        <receiver android:name=".receivers.WiFiConnectionSettingReceiver">
```

Comment:
Are these name updates in line with the corresponding android drivers

Reply:
I would rather keep service names to avoid unnecessary confusion between the old and new version of the service

Reply:
All broadcasts are called without Receiver name in:
- https://github.com/appium/appium-android-driver
- https://github.com/appium/appium-adb

Reply:
Yes, I would like to fix that. As a consequence we could wait until this PR is merged and published and then apply the necessary changes to appium-adb. See https://github.com/appium/appium-adb/pull/255